pkg/catalog/manager/manager_test.go

```diff
@@ -15,20 +15,20 @@ func TestLatestAvailableTemplateVersion(t *testing.T) {
 				{
 					ExternalID:        "catalog://?catalog=library&template=artifactory-ha&version=0.12.16",
 					Version:           "0.12.16",
-					RancherMinVersion: "v2.2.0",
-					RancherMaxVersion: "v2.3.0",
+					RancherMinVersion: "v2.2.0-0",
```

Comment:
Can you explain what the "-0" is for? I don't think the catalogs we use would include this in the constraints and I don't see you adding it to the versions in ValidateRancherVersion.

Reply:
Updated, do not need it anymore.